lldb/test/API/lang/swift/variables/generic_enums/TestSwiftGenericEnums.py

```diff
@@ -83,24 +59,21 @@ def cleanup():
         self.assertTrue(enumvar.GetValue() is None,
                         "static type has a value when it shouldn't")
         enumvar = enumvar.GetDynamicValue(lldb.eDynamicCanRunTarget)
-        self.assertTrue(
-            enumvar.GetValue() == "Some",
-            "dynamic type's value should be Some")
-        self.assertTrue(
-            enumvar.GetSummary() == "3",
+        # FIXME?
+        #self.assertEqual(
+        #    enumvar.GetValue(), "Some",
+        #    "dynamic type's value should be Some")
```

Comment:
what happened here?

Reply:
It returns 3 instead of `some`, which may or may not be correct. I'll file a bug. (This is the previously XFAILed test)

Reply:
interesting.

Reply:
I was under the impression that for Some,None enums, we always suppress the Some/None and either report nil or the value.  The only time you should see Some or None is if you do `--raw`.  Seems like if we're showing the Some that was not the desired behavior, so this test was testing a bug which has been fixed.

Reply:
But asserting that you get the value here is a useful test of this behavior.